policy_definitions/tagging/policy_assignment_qby_deploy_require_tags.tmpl.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "QBY-Deploy-Require-Tags",
```

Comment:
As this policy does not deploy anything. I think require tags is enough
```suggestion
    "name": "QBY-Require-Tags",
```

Reply:
Changed as suggested.